letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -319,6 +319,32 @@ else
   BOOTSTRAP_VERSION="Unknown 1"
 fi
 
+PrevBootstrapVersion() {
+  if [ -f $VENV_BOOTSTRAP_VERSION ]; then
+    cat $VENV_BOOTSTRAP_VERSION
+  elif grep -Fqx "$BOOTSTRAP_VERSION" << "UNLIKELY_EOF"
+NO_BOOTSTRAP 1
+BootstrapDebCommon 1
```

Comment:
OK I made this change. I treated the case where we don't know how to bootstrap on your system the same was as when we include `--no-bootstrap`. While this is unlikely to matter much, it stops us from writing "Unknown..." to disk and prevents us the following:

1. A user on an unsupported system runs Certbot with `--no-bootstrap`.
2. They later forget to include `--no-bootstrap`, we tell them they need to rebootstrap, and we exit with a warning saying we don't know how to bootstrap on your system.

We now properly preserve the bootstrap version when recreating the venv when `--no-bootstrap` is included on the command line.